README.md

```diff
@@ -47,7 +47,7 @@ python3 troi.py --help
 ```
 virtualenv -p python .ve
 .ve\Scripts\activate.bat
-pip install -r requirements.txt -r requirements_test.txt
+pip install .[tests]
 python troi.py --help
```

Comment:
We could now also run just `troi --help`, because the entrypoint enables us to do so.

@mayhem Should I update this? Do we want to remove the custom entrypoint?

Reply:
Same thing is up to discuss for the tests below

Reply:
Yes, lets do it.